jetbrains-core/src/software/aws/toolkits/jetbrains/utils/ui/UiUtils.kt

```diff
@@ -248,19 +250,37 @@ class ResizingTextColumnRenderer : ResizingColumnRenderer() {
  */
 fun CellBuilder<DialogPanel>.installOnParent(applies: () -> Boolean = { true }): CellBuilder<DialogPanel> {
     withValidationOnApply {
-        if (!applies()) {
-            null
-        } else {
-            val errors = this@installOnParent.component.validateCallbacks.mapNotNull { it() }
-            if (errors.isEmpty()) {
-                this@installOnParent.component.apply()
-            }
-            errors.firstOrNull()
-        }
+        validate(applies, it)
+    }
+    return this
+}
+
+fun Cell2<DialogPanel>.installOnParent(applies: () -> Boolean = { true }): Cell2<DialogPanel> {
+    validationOnApply {
+        validate(applies, it)
     }
     return this
 }
 
+private inline fun validate(applies: () -> Boolean, component: DialogPanel): ValidationInfo? =
+    if (!applies()) {
+        null
+    } else {
+        val errors = component.validateCallbacks.mapNotNull { it() }
+        if (errors.isEmpty()) {
+            component.apply()
+        }
+        errors.firstOrNull()
+    }
+
+// backport since removed in 223
```

Comment:
Could we change the comment to remove when min is 223

Reply:
this is more like remove when no longer used since this util doesn't exist anymore